rtb/norme_di_progetto_ver0.6.1.typ

```diff
@@ -531,9 +564,177 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+Al fine di coordinare in modo efficace le attività di gruppo, le discussioni di _routine_ avverranno su Telegram, mentre eventuali questioni critiche saranno affrontate durante incontri straordinari attraverso videochiamate su Discord.\
+
+===== Comunicazioni esterne
+Il responsabile del progetto sarà incaricato di gestire il dialogo esterno attraverso l'indirizzo _email_ "sweat.unipd\@gmail.com".
+Per quanto riguarda il mezzo di comunicazione per i SAL con i proponenti lo strumento utilizzato è Google Meet, che rende le videochiamate accessibili a tutti.
+
+==== Riunioni
+Al fine di garantire l'efficienza delle riunioni il responsabile corrente avrà il compito di interloquire con i proponenti per riepilogare i punti principali del periodo trascorso e esprimere dubbi e incertezze emerse dal gruppo.
+
+===== Riunioni Interne
+Le riunioni interne sono programmate di comune accordo tra i membri del gruppo. In caso di necessità, è possibile richiedere riunioni straordinarie durante la settimana tramite il canale dedicato su Telegram, con data e orario stabiliti attraverso un sondaggio. Tutte le riunioni _online_ si svolgeranno nel canale Discord appositamente designato. Nelle riunini interne l'obiettivo finale è quello di risolvere le mancanze segnalate dall'ordine del giorno, per cui nella fase finale della riunione, il responsabile, dopo aver discusso con il gruppo, pianifica le attività da svolgere e crea rispettivamente le _issue_ come da procedura sopra.
+
+===== Riunioni Esterne
+Le riunioni esterne coinvolgono i membri del gruppo e il proponente.\
+Per le riunioni con il proponente, viene utilizzata la piattaforma Google Meet, e l'indirizzo viene comunicato al _team_ di volta in volta. In accordo con l'azienda proponente gli incontri di fine _sprint_ avvengono ogni due settimane, in quanto lo _sprint_ dura due settimane, mentre a metà di ogni _sprint_ si svolgerà un incontro di allineamento più breve per un confronto e degli aggiustamenti.
+
+===== Compiti del responsabile
+- Esporre i punti all'ordine del giorno.
+- Fare il punto su quanto fatto e sul rispetto degli obiettivi posti.
+- Pianificare le prossime attività da svolgere.
+- Assegnare i _task_ ai membri del gruppo.
+
+==== Verbali
+===== Verbali Interni
+Al fine di tracciare le discussioni svolte, le decisioni prese dal gruppo e soprattutto le motivazioni dietro ad esse, al termine di ogni incontro interno al gruppo, viene aperta una _issue_ su Github per la preparazione, la verifica e l'approvazione del verbale. Il compito di redigere il verbale, seguendo il formato indicato nella sezione 3.1 di questo documento, è affidato al responsabile, il quale deve assicurarsi di includere tutte le informazioni rilevanti discusse.
```

Comment:
le motivazioni dietro ad esse... toglierei la virgola dopo e anche dopo "interno al gruppo".

Reply:
sulle NdP leggo questo: 
```
=== Riferimenti
==== Contestuali
Per riferirsi ad una sezione del documento per una spiegazione più dettagliata si adotta la dicitura "(sez. [I])" dove I indica l'indice della sezione (ad esempio "(sez. 3.2)").
```
quindi il riferimento al paragrafo mi sembra coerente non mettere il link, dammi feedback

Reply:
No. Va messo. Per quanto ne avevamo discusso in privato, prec conferma sul fatto di aggiungere il fatto che vada fatto il riferimento.